src/plugin/markers/index.js

```diff
@@ -137,14 +157,10 @@ export default class MarkersPlugin {
             time: params.time,
             label: params.label,
             color: params.color || DEFAULT_FILL_COLOR,
-            position: params.position || DEFAULT_POSITION
+            position: params.position || DEFAULT_POSITION,
+            draggable: !!params.draggable
         };
 
-        if (params.markerElement) {
-            this.markerWidth = params.markerElement.width;
```

Comment:
i could see this breaking backwards compatibility?

Reply:
I think not, that clause controls the position of custom markler elements, I've moved that logic to line 237, this allows to have a different width for each marker.

The current implementation has certain side effects, you can test it on the example [page](https://wavesurfer-js.org/example/markers/index.html), by clicking the marker v1.

Maybe this should be handle as a bug ?

Reply:
> Maybe this should be handle as a bug ?

If you mean it's a separate bug, please open a new ticket or pr for that.